File: app/src/main/java/mchehab/com/customview/MainActivity.java
package mchehab.com.customview;

import android.databinding.DataBindingUtil;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import mchehab.com.customview.databinding.ActivityMainBinding;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Person person = new Person("Mohamad", "Chehab", "https://www.w3schools" +
                ".com/w3css/img_lights.jpg", 23);
        ActivityMainBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_main);
        binding.setPerson(person);
    }
}

File: kotlinapp/src/main/java/mchehab/com/kotlinapp/MainActivity.kt
package mchehab.com.kotlinapp

import android.databinding.DataBindingUtil
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import mchehab.com.kotlinapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val person = Person("Mohamad", "Chehab", "https://www.w3schools.com/w3css/img_lights.jpg",
                23)
        val binding = DataBindingUtil.setContentView<ActivityMainBinding>(this, R.layout.activity_main)
        binding.person = person
    }
}